src/SMU-Agilent_B29xx/main.py

```diff
@@ -153,38 +160,39 @@ def configure(self):
             self.port.write(":SENSe:AVER:COUN 1")  
         """
 
-        self.port.write(":OUTP%s:PROT ON" % self.channel)    
```

Comment:
What was the purpose of OUTP:PROT ON? Enable output protection? Isn't it dangerous if that is removed?